src/components/PeoplePage.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useEffect, useState } from 'react';
+import { Loader } from './Loader';
+import { getPeople } from '../api';
+import { PersonType } from '../types';
+import { PeopleTable } from './PeopleTable';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<PersonType[]>([]);
+  const [error, setError] = useState(false);
+  const [isLoading, setIsLoadig] = useState(false);
+  const [noData, setNoData] = useState(false);
+
+  useEffect(() => {
+    getPeople()
+      .then((data) => {
+        if (data.length < 1) {
+          setNoData(true);
+        }
+
+        setPeople(data);
+        setIsLoadig(true);
```

Comment:
looks broken, you must set isLoading true before the request and set false in finally block